core/computing-unit-managing-service/src/main/scala/edu/uci/ics/texera/service/util/KubernetesClient.scala

```diff
@@ -142,7 +143,33 @@ object KubernetesClient {
       .endPort()
       .withEnv(envList)
       .withResources(resourceBuilder.build())
-      .endContainer()
+
+    // If shmSize requested, mount /dev/shm
+    shmSize.foreach { _ =>
+      containerBuilder
+        .addNewVolumeMount()
+        .withName("dshm")
```

Comment:
Is the name globally unique? If so, does this mean other CU will mount the same volume?

Reply:
Not global, this is isolated within each CU